packages/react-components/react-datepicker-compat/project.json

```diff
@@ -4,5 +4,5 @@
   "projectType": "library",
   "implicitDependencies": [],
   "sourceRoot": "packages/react-components/react-datepicker-compat/src",
-  "tags": ["vNext", "platform:web"]
+  "tags": ["vNext", "platform:web", "compat"]
```

Comment:
this compat package was create before `react-library` generator existed so we didnt have this convention in place. updating manually